### ISSUE TITLE
not compatible with 0.47 RN versions

### ISSUE THREAD

**Issue by kenma9123** [magus/react-native-facebook-login#258]
The build is failing on 0.47 and up RN versions.

```
:react-native-facebook-login:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
H:\Git\React-Native\iponChallenge\node_modules\react-native-facebook-login\android\src\main\java\com\magus\fblogin\FacebookLoginPackage.java:20: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-facebook-login:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.
```

Edit; it seems somebody fixed it with this commit #256

**Comment by Brhernann** [comment]
i have this problem too! 

please help!.

**Comment by kenma9123** [comment]
@Brhernann or for the time being use this PR #256 . Just go to the repo and use it as reference.

**Comment by dbyilmaz** [comment]
@kenma9123  is it solved ?

**Comment by chrusart** [comment]
It's solved, not merged tho.

**Comment by cinder92** [comment]
@kenma9123  just open file that Android Studio said, then comment, the method, and run again. i has same issue, but now its fixed.

This is because an update for native modules in Android.

**Comment by patw0929** [comment]
#256 had been merged and released as 1.6.0.
